File: 724-find-pivot-index/find-pivot-index.ts
function pivotIndex(nums: number[]): number {
    let total = 0
    for (let i = 0;i<nums.length;i++){
        total+=nums[i]
    }
    let pre = 0;
    for (let i = 0;i<nums.length;i++){
        if (pre ==total-(nums[i]+pre)){
            return i;
        }
        pre +=nums[i]
    }
    return -1;
};

File: 724-find-pivot-index/find-pivot-index.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        total = 0
        for i in nums:
            total+=i
        pre = 0
        for i in range(len(nums)):
            if pre == total-(nums[i]+pre):
                return i
            pre+=nums[i]
        return -1
